.github/workflows/main.yml

```diff
@@ -0,0 +1,18 @@
+name: housekeeping
+on: [push, pull_request]
+jobs:
+    tsc-and-format:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout code
+              uses: actions/checkout@v2
+
+            - name: Setup node
+              uses: actions/setup-node@v2
+              with:
+                  node-version: '14'
+            - name: Install dependencies
```

Comment:
This name could also be more descriptive as you also run two other commands after install

Reply:
Changed it to `Install dependencies and run tsc and prettier` in 1cbf73c.